src/coreclr/jit/emit.h

```diff
@@ -780,6 +780,9 @@ class emitter
         unsigned _idCallRegPtr : 1; // IL indirect calls: addr in reg
         unsigned _idCallAddr : 1;   // IL indirect calls: can make a direct call to iiaAddr
         unsigned _idNoGC : 1;       // Some helpers don't get recorded in GC tables
+#if defined(TARGET_XARCH)
+        unsigned _idEmbBroadcast : 1;
```

Comment:
We'll ultimately need space to encode:
* `EVEX.aaa`: Embedded opmask register specifier
* `EVEX.Z`: Zeroing/Merging
* `EVEX.b`: Broadcast/RC/SAE Context
* `EVEX.L'L`: Vector length/RC

This is 7 additional bits, so it might be good to make this be the more general `EVEX.b` bit that way it is usable for `Broadcast`, `RC`, and `SAE`. Maybe `_idEvexBContext` as a name? 

-- 
There are also a couple comments higher up that need to be fixed as well, discussing the number of bits being used in each area, etc.